lib/driver.js

```diff
@@ -568,23 +568,31 @@ class XCUITestDriver extends BaseDriver {
     }
   }
 
+  async translateDeviceName (pv, dn = '') {
```

Comment:
I'd prefer this to be a helper method, in `lib/utils.js` since it does not use any state off the driver object.